site/content/blog/terraform-carvel-provider.md

```diff
@@ -104,7 +104,7 @@ For readers following along, the working directory looks something like this at
 └── ytt-config
     ├── frontend.yaml
     ├── guestbook-all-in-one.yaml
-    └── redis-slave.yaml
```

Comment:
Good call out. I think this PR will land first but I expect the other repos to follow quickly so it should only be a brief time (a day?) where they're out of sync. _I think_ that's an acceptable risk but please let me know if you feel otherwise.

Reply:
Slight correction: I think the first PR to land is this one: https://github.com/vmware-tanzu/carvel-release-scripts/pull/27. Then _this_ one. Then the other repos.

Reply:
As an update, I removed this change for now so that we can make a proper fix in the future. I captured the work here: https://github.com/vmware-tanzu/terraform-provider-carvel/issues/31